site/content/cluster/binaries_cluster.md

```diff
@@ -0,0 +1,264 @@
+{{% /notice %}}
+
+## Download and Install a Binary
+
+You can download the latest release as [pre-compiled binaries from the M3 GitHub page](https://github.com/m3db/m3/releases/latest). Inside the expanded archive are binaries for `m3dbnode`, which combines a coordinator and storage node, and a binary for `m3coordinator`, which is a standalone coordinator node.
+
+## Build from Source
+
+### Prerequisites
+
+-   [Go 1.10 or higher](https://golang.org/dl/)
+-   [Make](https://www.gnu.org/software/make/)
+
+### Build
+
+[Clone the codebase](https://github.com/m3db/m3) and run `make m3dbnode` to generate a binary for a combination coordinator and storage node, or `make m3coordinator` to generate a binary for a standalone coordinator node.
+
+## Provision a Host
+
+Enough background, let's create a real cluster!
+
+M3 in production can run on local or cloud-based VMs, or bare-metal servers. M3 supports all popular Linux distributions (Ubuntu, RHEL, CentOS), and [let us know](https://github.com/m3db/m3/issues/new/choose) if you have any issues with your preferred distribution.
+
+### Network
+
+{{% notice tip %}}
+If you use AWS or GCP, we recommend you use static IPs so that if you need to replace a host, you don't have to update configuration files on all the hosts, but decommission the old seed node and provision a new seed node with the same host ID and static IP that the old seed node had. If you're using AWS you can use an [Elastic Network Interface](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-eni.html) on a Virtual Private Cloud (VPC) and for GCP you can use an [internal static IP address](https://cloud.google.com/compute/docs/ip-addresses/reserve-static-internal-ip-address).
+{{% /notice %}}
+
+This example creates three static IP addresses for three storage nodes, using the embedded coordinator.
+
+This guide assumes you have host names configured, i.e., running `hostname` on a host in the cluster returns the host ID you use when creating the cluster placement.
+
+{{% notice tip %}}
+When using GCP the name of your instance is the host name. When you create an instance, click _Management, disks, networking, SSH keys_, under _Networking_, click the default interface, click the _Primary internal IP_ drop down, select _Reserve a static internal IP address_, give it an appropriate name and description, and use _Assign automatically_.
+{{% /notice %}}
+
+{{% notice tip %}}
+When using AWS, you can use the host name supplied for the provisioned VM as your host ID, or use the `environment` host ID resolver and pass the host ID when launching the database process with an environment variable.
+{{% /notice %}}
+
+For example, if you used `M3DC_HOST_ID` for the environment variable name, use the following in your configuration:
+
+```yaml
+hostID:
+  resolver: hostname
+    - ${M3DC_HOST_ID:""}
```

Comment:
I believe this should be:
```
hostID:
  resolver: config
  value: ${M3DC_HOST_ID:""}
```

Reply:
it's the current instructions here - https://m3db.io/docs/how_to/cluster_hard_way/ Any idea why you say it should be different?